controllers/articles.js

```diff
@@ -239,6 +239,150 @@ const articles = {
       }
     }
   },
+
+  /*
+  * Bookmarking and article so that one can come back
+  * and read it later.
+  */
+  createBookmark: async (req, res) => {
+    try {
+      const { slug } = req.params;
+      const { id: userId } = req.user;
+      if (!userId) {
+        res.status(401).send({
+          status: res.statusCode,
+          errorMessage: 'Please first login to bookmark this article',
+        });
+      }
+      const findArticle = await article.findOne({ where: { slug } });
```

Comment:
Would you please use descriptive names on variables? `i.e findArticle sounds like a name of a function to find an article while it's actually an article`